peachjam/admin.py

```diff
@@ -322,11 +322,6 @@ def clean_content_html(self):
             return self.instance.content_html
         return self.cleaned_data["content_html"]
 
-    def _save_m2m(self):
-        super()._save_m2m()
-        # update document text
-        self.instance.update_text_content()
-
```

Comment:
@longhotsummer this is getting called before the source file is saved and updated so we can move it to after the save_related method